package.json

```diff
@@ -40,6 +43,8 @@
     "not op_mini all"
   ],
   "devDependencies": {
+    "@material-ui/pickers": "^3.2.10",

```

Comment:
These probably shouldn't be devDependencies; I think your production build (`npm run build`) will result in an incomplete output here.